ci.rb

```diff
@@ -66,6 +66,8 @@
 options = {}
 options[:delay_after_run] = 300
 options[:maximum_branch_age] = 30
+options[:verbose] = false
+options[:keep_build_folder] = false
```

Comment:
OK, so for these particular changes, we don't have to adjust the decent_ci_runner command line or config file if we don't want to; it will have these defaults in place.  Right?

Reply:
Correct.